pkg/check/withdraw/withdraw.go

```diff
@@ -0,0 +1,81 @@
+package withdraw
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethersphere/beekeeper/pkg/beekeeper"
+	"github.com/ethersphere/beekeeper/pkg/logging"
+	"github.com/ethersphere/beekeeper/pkg/orchestration"
+	test "github.com/ethersphere/beekeeper/pkg/test"
+)
+
+// Options represents check options
+type Options struct {
+	TargetAddr string
+}
+
+// NewDefaultOptions returns new default options
+func NewDefaultOptions() Options {
+	return Options{}
+}
+
+// compile check whether Check implements interface
+var _ beekeeper.Action = (*Check)(nil)
+
+// Check instance
+type Check struct {
+	logger logging.Logger
+}
+
+// NewCheck returns new check
+func NewCheck(logger logging.Logger) beekeeper.Action {
+	return &Check{
+		logger: logger,
+	}
+}
+
+func (c *Check) Run(ctx context.Context, cluster orchestration.Cluster, opts interface{}) (err error) {
+	o, ok := opts.(Options)
+	if !ok {
+		return fmt.Errorf("invalid options type")
+	}
+
+	var checkCase *test.CheckCase
+
+	if checkCase, err = test.NewCheckCase(ctx, cluster, test.CaseOptions{}, c.logger); err != nil {
+		return err
+	}
+
+	target := checkCase.Bee(1)
+
+	c.logger.Infof("target is %s", target.Name())
+
+	c.logger.Info("withdrawing native...")
+
+	if err := target.Withdraw(ctx, "NativeToken", o.TargetAddr); err != nil {
```

Comment:
the before and after balances should be compared after the withdraw

Reply:
it's being tested inside the function, this way I can use the same method for different coin/addresses:

https://github.com/ethersphere/beekeeper/pull/386/files#diff-26ffebd2abeeaff0e1d88eab9a47503856f31fcfc6fd6704b431ea5c4b72727cR125